pages/linux/logger.md

```diff
@@ -0,0 +1,23 @@
+# logger
+
+> Add messages to syslog.
+
+- Log a message to syslog:
+
+`logger {{message}}`
+
+- Take input from stdin and log to syslog:
+
+`tail -f {{app.log}} | logger`
```

Comment:
Hmm... maybe something like `echo {{"log entry"}}` would make it clearer, then. Or using cat instead of tail -f, and perhaps a path in the token (something like path/to/log_file). Otherwise it doesn't seem particularly generic. What do you think?